app/assets/javascripts/Components/Result/submission_selector.jsx

```diff
@@ -140,3 +144,25 @@ export class SubmissionSelector extends React.Component {
     );
   }
 }
+
+SubmissionSelector.propTypes = {
+  assignment_max_mark: PropTypes.number,
```

Comment:
@WeissSa this prop type is logically correct, but currently is flagged as an error in the actual web application. Why don't you dig into this a little bit and modify the React components as necessary to ensure the prop value is always a number?

Reply:
Should be fixed I think.